samcli/local/apigw/service.py

```diff
@@ -366,6 +371,37 @@ def _construct_event(flask_request, port, binary_types):
         LOG.debug("Constructed String representation of Event to invoke Lambda. Event: %s", event_str)
         return event_str
 
+    @staticmethod
+    def _query_string_params(flask_request):
+        """
+        Constructs an APIGW equivalent query string dictionary
+
+        Parameters
+        ----------
+        flask_request request
+            Request from Flask
+
+        Returns dict (str: str)
+        -------
+            Empty dict if no query params where in the request otherwise returns a dictionary of key to value
+
+        """
+        query_string_dict = {}
+
+        # Flask returns an ImmutableMultiDict so convert to a dictionary that becomes
+        # a dict(str: list) then iterate over
+        for query_string_key, query_string_list in dict(flask_request.args).items():
+            query_string_value_length = len(query_string_list)
```

Comment:
nit: This variable is unnecessary. You can just do `if query_string_list`to know if the list is empty or not